llarp/rpc/rpc.cpp

```diff
@@ -271,6 +278,10 @@ namespace llarp
       HandleJSONRPC(Method_t method,
                     ABSL_ATTRIBUTE_UNUSED const Params& params) override
       {
+        if(method == "llarp.admin.start")
+        {
+          return StartRouter();
+        }
```

Comment:
this should be `llarp.admin.wakeup` so we can have a `llarp.admin.suspend` as well

Reply:
I was thinking start + stop, versus wakeup. will defer to you though